tgui/packages/tgui/interfaces/AresAdmin.js

```diff
@@ -256,6 +256,18 @@ const MainMenu = (props, context) => {
               onClick={() => act('page_requisitions')}
             />
           </Stack.Item>
+          <Stack.Item>
+            <Button
+              content="Tech Control Log"
+              tooltip="Review the Intel Tech Log."
+              icon="magnifying-glass-chart"
+              ml="auto"
+              px="2rem"
+              width="25vw"
+              bold
+              onClick={() => act('page_tech')}
+            />
+          </Stack.Item>
```

Comment:
This button just crashes the panel (the remote ares panel). You don't pass admin_login as data to this.
![image](https://github.com/cmss13-devs/cmss13/assets/76988376/db0d6851-8620-4c4a-b3eb-ecb20137cda7)